modules/dailymotionBidAdapter.js

```diff
@@ -47,7 +47,7 @@ export const spec = {
    * @return boolean True if this is a valid bid, and false otherwise.
    */
   isBidRequestValid: function (bid) {
-    return (typeof bid.params.apiKey === 'string');
+    return ((typeof bid !== 'undefined') && (typeof bid?.params?.apiKey === 'string') && (bid.params.apiKey.length > 10));
```

Comment:
return typeof bid?.params?.apiKey === 'string' && bid.params.apiKey.length > 10;

Reply:
I tried but it still produced error when no bid is passed

Reply:
typo in your test,
You can try when launching node
`undefined?.undefined?.undefined` which will produce `undefined`